test/assignment7-buildroot/assignment-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# 1st argument: absolute or relative path to the base directory
+
+cd `dirname $0`
+
+source script-helpers
+
+script_dir="$(pwd -P )"
+testdir=$1
+qemu_executable_path=/usr/bin	#Path where writer,finder,tester.sh are stored
+
+# Invoke docker script with --env SKIP_BUILD=1 --env DO_VALIDATE=1 to perform a validation only
+# test
+echo "starting test with SKIP_BUILD ${SKIP_BUILD} and DO_VALIDATE ${DO_VALIDATE}"
+if [[ -z ${SKIP_BUILD} || ${SKIP_BUILD} -eq 0 ]]; then
+    . ./assignment-7-build.sh ${testdir}
```

Comment:
@akshita-bhasin can we just call buildroot-common-build here instead of going through the extra script?

Reply:
Yes!